lib/tasks/learn_to_rank.rake

```diff
@@ -98,7 +99,7 @@ namespace :learn_to_rank do
   end
 
   def assert_ltr!
-    raise "set $ENABLE_LTR to use learn_to_rank" if ENV["ENABLE_LTR"].nil?
+    raise 'set $ENABLE_LTR to "true" to use learn_to_rank' if !Search::RelevanceHelpers.ltr_enabled?
```

Comment:
Surprised the linter didn't complain about using `if !` instead of `unless`

Reply:
Yeah, weird... is the Rakefile not included.  Seems like a reasonable adjustment.  Incoming